src/tasks/level10.ts

```diff
@@ -119,6 +122,8 @@ export const GiantQuest: Quest = {
         else
           return {
             modifier: "-combat, item",
+            // eslint-disable-next-line libram/verify-constants
+            equip: $items`bat wings`,
```

Comment:
This one is on me, but this doesn't _require_ bat wings right? Just "equip if possible" yes?

Reply:
Right, no hard requirement.